scene/main/node.cpp

```diff
@@ -1388,6 +1396,11 @@ bool Node::is_greater_than(const Node *p_node) const {
 	return res;
 }
 
+bool Node::is_priority_than(const Node *p_node) const {
```

Comment:
`is_priority_more_tasty_than`? :P I'd suggest `is_priority_higher_than` or something similar, the current name is confusing :sweat_smile:

Reply:
Yes better :D